Cesium3DTilesSelection/src/ViewState.cpp

```diff
@@ -1,14 +1,64 @@
 #include "Cesium3DTilesSelection/ViewState.h"
 
+#include "Cesium3DTilesSelection/BoundingVolume.h"
+#include "CesiumGeometry/BoundingSphere.h"
+#include "CesiumGeometry/CullingResult.h"
+#include "CesiumGeometry/OrientedBoundingBox.h"
+#include "CesiumGeospatial/BoundingRegion.h"
+#include "CesiumGeospatial/BoundingRegionWithLooseFittingHeights.h"
+#include "CesiumGeospatial/Cartographic.h"
+#include "CesiumGeospatial/Ellipsoid.h"
+#include "CesiumGeospatial/S2CellBoundingVolume.h"
+
 #include <CesiumGeometry/CullingVolume.h>
 
+#include <glm/common.hpp>
+#include <glm/ext/vector_double2.hpp>
+#include <glm/ext/vector_double3.hpp>
 #include <glm/trigonometric.hpp>
 
+#include <optional>
+#include <variant>
+
 using namespace CesiumGeometry;
 using namespace CesiumGeospatial;
 
 namespace Cesium3DTilesSelection {
 
+namespace {
```

Comment:
Better to leave this code where it was in the file, and just change it from `static` to `namespace { ... }`.